### DISCUSSION TITLE
Unable to Reset GitHub Password and Access Account.

### DISCUSSION THREAD

**Discussion by rabs-ramii-801** [community/community#137489] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hello,

I am experiencing an issue with my GitHub account (username: @rabs-ramii).

On August 27, 2024, I received an email from GitHub notifying me of a suspicious login attempt on my account, which led to a forced password reset. I successfully reset my password and accessed my account without any issues.
However, on August 31, 2024, I tried to sign in again using my latest password, but I received an error message stating "Incorrect username, email, or password." I attempted to reset my password again using the "Forgot password" option, but I did not receive any password reset email, even after multiple attempts and checking my spam/junk folder.

I also tried to contact GitHub Support via email, but my message was blocked due to organizational settings. I am currently unable to access my account and am concerned that it may have been compromised.

Could someone please assist me in regaining access to my account?

Thank you for your help.
#account, #password-reset, #security, #account-recover, #help, #blocked

**Comment by Dilshan-Mindika** [comment]
Check your email settings
Ensure that your email provider isn't filtering out GitHub's emails. Sometimes, security settings or email filters can prevent password reset emails from arriving in your inbox.

**Reply by rabs-ramii-801** [reply]
Hello Dilshan-Mindika,

Thank you for your suggestion. I’ve checked my email settings, and everything appears to be configured correctly. I haven’t found any filters or blocks that could be preventing GitHub’s emails from reaching me.

I’ve also tried various troubleshooting steps, such as checking my spam/junk folder and verifying that my email address is correct. Unfortunately, I am still not receiving the password reset emails and am unable to access my GitHub account.

Could you please provide further assistance or alternative solutions to regain access to my account? I’m concerned that there might be a deeper issue at play.

Thank you for your help!

Best regards,
rabs-ramii-801

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @rabs-ramii-801 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
